app/models/validation.rb

```diff
@@ -0,0 +1,19 @@
+class Validation < ApplicationRecord
+  belongs_to :applicant
+
+  enum status: [ :started, :in_process, :complete, :error ]

```

Comment:
One pattern I try to enforce is ALWAYS using hashes for enums. This provides a strict mapping for the enum, and assures that values don't change in the future.

Reply:
Changed to hash, thanks